staging/vhost-device-sound/src/audio_backends/pipewire.rs

```diff
@@ -299,12 +299,25 @@ impl AudioBackend for PwBackend {
 
             let mut param = [Pod::from_bytes(&values).unwrap()];
 
+            let spa_dir: spa::Direction = stream_params[stream_id as usize].direction.into();
```

Comment:
It looks like `spa_dir` isn't used for anything other than `match`ing, right? It is [inadequately modeled in the bindings](https://gitlab.freedesktop.org/pipewire/pipewire-rs/-/blob/26ed5ffd8e95ddde923bc09a2174127c8e12249e/libspa/src/utils/direction.rs#L7) as a single tuple struct even though [`enum spa_direction`](https://docs.pipewire.org/group__spa__utils__defs.html#ga9f2fae4a58cc2e3fdf568d1afc9dc7c1) has two possible values: 
`SPA_DIRECTION_INPUT` `SPA_DIRECTION_OUTPUT`

I suggest using the crate's `Direction` instead to avoid the needless panicking match patterns.